resolwe_bio/processes/variant_calling/gatk.yml

```diff
@@ -217,7 +217,7 @@
         --stand-call-conf {{ stand_call_conf }} \
         --max-reads-per-alignment-start {{ max_reads }} \
         --dbsnp {{ dbsnp.vcf.file }} \
-        -L ${INTERVALS} \
+        {% if intervals %} -L ${INTERVALS} {% endif %} \
```

Comment:
This process has `intervals` and `intervals_bed` inputs, to handle two use-cases. You need to account for both options.